JavadocStyleGuide.md

```diff
@@ -0,0 +1,242 @@
+- Insert a blank comment line between the description and the list of tags, as shown.
+- The first line that begins with an `@` character ends the description; you cannot continue the description following block tags.
+- Block tags must be added in order.
+- The last line contains the end-comment delimiter ( `*/`).
+
+So lines won't wrap, limit any doc-comment lines to 80 characters.
+
+### 1.1.2 HTML
+
+Javadoc permits only a subset of HTML tags: `"a", "abbr", "acronym", "address", "area", "b", "bdo", "big", "blockquote", "br", "caption", "cite", "code", "colgroup", "dd", "del", "dfn", "div", "dl", "dt", "em", "fieldset", "font", "h1", "h2", "h3", "h4", "h5", "h6", "hr", "i", "img", "ins", "kbd", "li", "ol", "p", "pre", "q", "samp", "small", "span", "strong", "sub", "sup", "table", "tbody", "td", "tfoot", "th", "thead", "tr", "tt", "u", "ul", "var"`.
+
+Attempt to limit your usage of HTML to: `"h1", "h2", "h3", "h4", "em", "li", "ol", "p", "ul"`.
+
+Some HTML element types allow authors to omit end tags: `<p>` and `<li>`.
+
+### 1.1.3 Paragraphs
+
+One blank line—that is, a line containing only the aligned leading asterisk (`*`)—appears between the last paragraph and before the group of block tags if present:
+
+```java
+/**
+ * that draw the image will incrementally paint on the screen.
+ *
+ * @param  url  an absolute URL giving the base location of the image
+ */
+```
+
+Each paragraph is denoted by a `<p>` which is placed on a separate line:
+
+```java
+/**
+ * argument is a specifier that is relative to the url argument.
+ * <p>
+ * This method always returns immediately, whether or not the
+ */
+```
+
+HTML tags for other block-level elements, such as `<ul>` or `<table>`, are not preceded with `<p>`.
+
+### 1.1.4 Symbols
+
+Entities for the less than symbol (<) and the greater than symbol (>) should be written as `&lt`; and `&gt;`. 
+Similarly, the ampersand (&) should be written as `&amp;`:
+
+```java
+/**
+ * This &amp; that.
+ */
+```
+
+### 1.1.5 Links
+
+The `{@link <class or method reference>}` tag is specifically used to link to the Javadoc of other classes and methods:
+
+```java
+/**
+ * The url argument must specify an absolute {@link classname}. The name
+ */
+```
+
+To add a reference to an external URL, use `<a href="URL#value">label</a>`.
+This adds a link as defined by `URL#value`. The `URL#value` is a relative or absolute URL:
+
+```java
+/**
+ * This is a link to <a href="http://www.google.com">Google</a>. The name
+ */
+```
+
+### 1.1.6 Code keywords, names, and variables
+
+Use `<code>...</code>` or `{@code ... }` style for keywords and names including:
+- Java keywords
+- package names
+- class names
+- method names
+- interface names
+- field names
+- argument names
+- code examples (see section 1.3 below)
+
+For example:
+
+```java
+/**
+ * Use the <code>Project</code> instance to configure the project.
+ * Use the {@code Project} instance to configure the project.
```

Comment:
I don't think this example is what we want to emulate.  I think we should favor the briefer `{@code SOMECODE}` syntax most of the time, except when special characters make that difficult, or we're doing a multiline snippet.  I think we need an example of a multiline snippet here.  

Why wouldn't `{@code Project}` use `@link` instead?

Reply:
@code just formats the text and @link links to an existing class in the remainder of your javadoc. @link is always prefered but if there is nothing to "link" to or if you want to highlight something like a very generic class you can use @code.
I updated the styleguide accordingly.